### ISSUE TITLE
Review and help with leaf temp and conductance datasets with time data and covariates

### ISSUE THREAD

**Issue by NewcombMaria** [terraref/reference-data#202]
@gsrohde I've formatted 3 spreadsheets that are probably close to being ready for Betydb using the API upload process (because they have time data).  I'll need your help reviewing to see if they are in an appropriate format for CSV trait data files.  Right now they are xlsx files in the Google directory 

https://drive.google.com/drive/folders/1TG5_qHqssuAzzAPBbvTC7DnEvxCIxFJY .  

I've highlighted in yellow columns that should be uploaded. The 3 spreadsheets differ by the hand-held instruments used to take measurements (ApogeeIRT, FlirTG167, and StomatalConductance), and all 3 datasets are associated with the same method.  

Specific questions: 
1)  Are the names/values/formats correct for:  citation, treatment, access_level, and method? 
2)  How are covariates connected to the appropriate trait variable?  2 of the spreadsheets have covariates, connected to traits as follows:  The FlirTG167 spreadsheet has 1 trait (surface_temperature_leaf) and 1 covariate (image_id) associated with the trait.  The StomatalConductance spreadsheet has 1 trait (leaf_stomatal_conductance) and 4 covariates associated with the trait (ambient_air_temperature, sensor_calibration, leaf_sensor_RH, and filter_sensor_RH). 
3) The time data in the ApogeeIRT file is only approximate to within about +/- 3 minutes.  Is there a way to account for that? 
4)  Associated images, for the FlirTG167 surface_temperature_leaf data.  I've included the image name in the notes column, and I also made the numerical component a covariate.  Will this work?  This might be a question for @dlebauer .  The images for each date are in 7 folders named by date, and these are in the same Google directory.  The plan as I understand it is to have the images stored in Clowder. 

Thanks for your help.  Let me know if you need more info from me.

**Comment by NewcombMaria** [comment]
@gsrohde there's another spreadsheet that I've formatted the same way, and I've just finished naming/defining 4 trait variables and 1 covariate that is a covariate to be associated wtih all 4 of the trait variables.  The xlsx spreadsheet is called Sorghum season4_Dualex_August_StressPhotosynthesis at the directory https://drive.google.com/drive/folders/1MVkQpJ8f_gCHZP17j63tXaAGCT1R5tAO. 

Are you available to link the covariate with the associated trait variables, and review the formatting?  So far there are 3 datasets referred to in this issue that have covariates.  This most recent dataset for the Dualex instrument also has local_datetime info.
